internal/testing/e2e/common.go

```diff
@@ -162,15 +162,15 @@ func (om *om) MMFConfigHTTP() *pb.FunctionConfig {
 // that parsing bugs can't hide logic bugs.
 const registrationInterval = time.Millisecond * 200
 const proposalCollectionInterval = time.Millisecond * 200
-const pendingReleaseTimeout = time.Millisecond * 200
```

Comment:
Do the backfill tests need a larger timeout? Given that this behavior may impact both backfill / non backfill scenarios - whats the impact of this on the regular e2e tests etc.? 

(I see this being used in TestTicketReleaseByTimeout / TestReleaseTickets etc.)

Reply:
@yeukovichd addressed this in Slack before:
>The reason why TestProposedBackfillUpdate is failing time to time because sometimes it takes a little bit longer than pendingReleaseTimeout (200ms in tests) to create\update backfill after tickets added to pending release in synchroniser.